pkgs/development/compilers/ghc/8.10.7.nix

```diff
@@ -99,11 +99,14 @@ let
 
     DYNAMIC_GHC_PROGRAMS = ${if enableShared then "YES" else "NO"}
     INTEGER_LIBRARY = ${if enableIntegerSimple then "integer-simple" else "integer-gmp"}
+    GhcLibWays = v
+  '' + lib.optionalString enableProfiledLibs ''
+    GhcLibWays += p
+  '' + lib.optionalString enableShared ''
+    GhcLibWays += dyn
   '' + lib.optionalString (targetPlatform != hostPlatform) ''
     Stage1Only = ${if targetPlatform.system == hostPlatform.system then "NO" else "YES"}
     CrossCompilePrefix = ${targetPrefix}
-  '' + lib.optionalString (!enableProfiledLibs) ''
-    GhcLibWays = "v dyn"
```

Comment:
Can you explain why you are using `v` unconditionally and adding `p` which apparently wasn’t present before?